website/content/docs/api-gateway/install.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Consul API Gateway Install
+description:  >-
+  Installing Consul API Gateway
+---
+
+# Installing Consul API Gateway
+
+This topic describes how to use the Consul API Gateway add-on module. It includes instructions for installation and configuration.
+
+## Requirements
+
+Ensure that the environment you are deploying Consul API Gateway in meets the requirements listed in the [Technical Specifications][tech-specs]. This includes validating that the requirements for minimum versions of software are met. See the [Release Notes][rel-notes] for the version of API Gateway you are deploying.
+
+## Installation
+
+1. Set the version of Consul API Gateway you are installing as an environment variable. The following steps use this environment variable in commands and configurations.
+
+```shell-session
```

Comment:
let's indent these code blocks, they're not nested under each bullet point. as a result, the numbered list isn't rendering correctly

<img width="939" alt="image" src="https://user-images.githubusercontent.com/7556325/182291170-36750d84-5f19-4682-92a5-80dfea73aa6f.png">